blog/_posts/2018-06-13-accessible-scala.md

```diff
@@ -32,7 +32,9 @@ We created an [online demo]. You can try it now! (Tip: It works best on google-c
 
 We also created a [vscode extension] so you can try on your project. Search for `Accessible Scala` in the 
 extension manager
-  
+
+<iframe style="margin: 20px auto; display: block;" width="560" height="315" src="https://www.youtube.com/embed/Up2ytnrsX6s" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
The video is impressive but I’m wondering if it’d better showing a piece of an actual program rather than a list of everything that’s supported.

Reply:
great suggestion, I'm on it :)